brave/browser-laptop
# Permission host mismatch

Fixes: https://github.com/brave/browser-laptop/issues/13471
Depends: https://github.com/brave/muon/pull/540

## Test plan
1. Open https://google.com
2. Change the address in the URL bar to sms://ggle.com. Verify the external permission message is for  sms://ggle.com and not https://google.com
3. Launch a webserver with test.html:
`<a href="sms://ggle.com">Test</a>`
4. Open http(s)://<webserver>/test.html
5. Click on the hyperlink, verify the permission is requested for `sms://ggle.com` and not `<webserver>`
